src/python/pants/backend/project_info/BUILD

```diff
@@ -1,14 +1,66 @@
-# Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
I intentionally put all rule code in the top level `backend/project_info`, rather than `backend/project_info/rules`. Why? In a V2-only world, it wouldn't make sense to have `rules/` anymore - they should just be top-level.